### PR TITLE
Enabling cloudfront logs

### LINKED ISSUE #487: Enable logging on Cloudfront distribution
## Type of Issue:
New feature

## Background
It was reported as a finding in securityhub that cloudfront distributions should have logging turn on.

## Proposed Approach
The approach is to create an s3 bucket to host the cloudfront logs and then enable logging on the cloudfront distribution in the quickstart repo to write its logs to the s3 bucket

## Value
This will help us resolve all the vulnerability findings pertaining to this issue.

## AC:
S3 bucket will be created to host the logs
Cloudfront logs will be enabled and the logs will be sent to the s3 bucket

### PR DESCRIPTION
**Pull request by joeaero19**
## Purpose
Enabling logging on clodfront distribution to write the logs to an s3 bucket
_Describe the problem or feature in addition to a link to the issues._

#### Linked Issues to Close

Closes #487 
## Approach
The approach is to create an s3 bucket to host the cloudfront logs and then enable logging on the cloudfront distribution in the quickstart repo to write the logs to the s3 bucket

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Review by shammons9281 — DISMISSED** [review]
Looks good to me.

File: services/ui/serverless.yml [hunk 1]
```diff
@@ -88,6 +88,32 @@ resources:
               Principal:
                 CanonicalUser: !GetAtt CloudFrontOriginAccessIdentity.S3CanonicalUserId
         Bucket: !Ref S3Bucket
+    LoggingBucket:
+      Type: "AWS::S3::Bucket"
+      Properties:
+        BucketName: !Sub ${self:service}-${self:custom.stage}-cloudfront-logs
```
Comment (dwhitecl) [inline services/ui/serverless.yml:94]: we should append the account ID to the end of this bucket as the bucket is a global name space and as folks fork this repo or pull this change in to address their own products they'd very likely have a naming collision
https://github.com/CMSgov/macpro-quickstart-serverless/blob/master/services/uploads/serverless.yml#L100
See above as an example

**Review by dwhitecl — CHANGES_REQUESTED** [review]

File: services/ui/serverless.yml (hunk 1 above)
Comment (shammons9281) [reply services/ui/serverless.yml:94]: good catch, thanks

Comment (joeaero19) [reply services/ui/serverless.yml]: Requested change has been applied. Thanks!

**Review by dwhitecl — APPROVED** [review]

**Pull request merged**